examples/playlist/README.md

```diff
@@ -0,0 +1,57 @@
+# Playlist examples
+This package explores a few options for implementing a playlist kind with kindsys.
+
+Specifically, this implements three options:
+1. Schemas defined in cue using thema
+2. Direct golang validation
+3. Schemas (and validation) based on jsonschema
+
+
+### Why playlist?  
```

Comment:
I'm confused; haven't we agreed that dashboards are THE EXAMPLE? I'm happy with any, but I'm just wondering why sometimes we kinda agree on kind as the use case to focus on as a great example, and why sometimes we agree on another one 🤷🏻

Reply:
dashboards -- with nested schemas at multiple versions -- are the reason/motivation/target for a complex schema system; specifically the "composable kinds"

This example is closer to a hello world -- we need to be grounded in something concrete to work against.  I have not yet seen any example try more than one version, so starting with something simple seems reasonable.